src/sentry/api/serializers/models/userreport.py

```diff
@@ -12,34 +12,40 @@ def get_attrs(self, item_list, user):
         queryset = list(EventUser.objects.filter(
             id__in=[i.event_user_id for i in item_list],
         ))
+
         event_users = {e.id: d for e, d in zip(queryset, serialize(queryset, user))}
 
+        events_list = Event.objects.filter(
+            project_id__in=[i.project_id for i in item_list],
```

Comment:
Can we cheat on this? Won't these all already be within the same project_id scope? This is going to be scoped within a single `Group`, which means that all `UserReport` objects are going to be within the same project.

We can cheat this a bit and just do:

```
project_id=item_list[0].project_id
```

Otherwise, we're sending an `IN` with a bunch of the same ids. At minimum, let's reduce this down to a set so it only sends uniques.

```
project_id__in={i.project_id for in in item_list}
```

Which will likely only send the SQL `IN (1)` instead of `IN (1, 1, 1, 1, 1, 1, 1)`